testcases/kernel/syscalls/brk/brk01.c

```diff
@@ -118,13 +118,13 @@ int main(int ac, char **av)
 
 		if (TEST_RETURN == -1) {
 
-			aft_brk_val = (long)sbrk(0);
+			aft_brk_val = (unsigned long)sbrk(0);
 			tst_resm(TFAIL | TTERRNO,
 				 "brk(%ld) failed (size before %ld, after %ld)",
 				 nbrkpt, cur_brk_val, aft_brk_val);
```

Comment:
You're introducing undefined behavior now. %ld is not valid conversation specifier for unsigned long type.